client/src/main/java/client/control/ClientControl.java

```diff
@@ -0,0 +1,31 @@
+package client.control;
+
+import java.io.IOException;
+
+public class ClientControl {
+
+    private final ClientConnection clientConnection;
+
+    public ClientControl() throws IOException {
+        this.clientConnection = new ClientConnection();
+        clientConnection.startConnection();
+    }
+
+    void input(ClientControlCommandType clientControlCommandType) {
```

Comment:
Not appropriate method name

1) not a verb
2) not only processing input but also sends messages